RFM69 gateway - unused function warning

```
In file included from ./MySensors.h:343:0,
                 from examples_linux/mysgw.cpp:83:
./drivers/RFM69/new/RFM69_new.cpp:663:12: warning: ‘void RFM69_ATCmode(bool, int16_t)’ defined but not used [-Wunused-function]
 LOCAL void RFM69_ATCmode(const bool onOff, const int16_t targetRSSI)
            ^
```
See forum:
https://forum.mysensors.org/topic/8933/mysensors-2-2-0-released/8

--- a/hal/transport/MyTransportRFM69.cpp
+++ b/hal/transport/MyTransportRFM69.cpp
@@ -26,8 +26,10 @@
 bool transportInit(void)
 {
 	const bool result = RFM69_initialise(MY_RFM69_FREQUENCY);
-#if !defined(MY_GATEWAY_FEATURE) && !defined(MY_RFM69_ATC_MODE_DISABLED)
-	// only enable ATC mode nodes
+#if defined(MY_GATEWAY_FEATURE) || defined(MY_RFM69_ATC_MODE_DISABLED)
+	// ATC mode disabled on GW
+	RFM69_ATCmode(false);
+#else
 	RFM69_ATCmode(true, MY_RFM69_ATC_TARGET_RSSI_DBM);
 #endif
 

